processing/src/test/java/org/apache/druid/query/aggregation/constant/LongConstantBufferAggregatorTest.java

```diff
@@ -38,9 +37,7 @@ public void setup()
   {
     randomVal = RandomUtils.nextLong();
     aggregator = new LongConstantBufferAggregator(randomVal);
-    byteBuffer = EasyMock.mock(ByteBuffer.class);
-    EasyMock.replay(byteBuffer);
-    EasyMock.verifyUnexpectedCalls(byteBuffer);
+    byteBuffer = ByteBuffer.allocate(Long.BYTES);
```

Comment:
I think the original test was trying to verify that the aggregator did not make any calls to the buffer. In that case setting `byteBuffer = null` accomplishes the original goal without using EasyMock.

Reply:
right, that may have been the intent, but it wasn't actually testing for anything, since it would call verify before actually running the test. I can make it null to ensure we test it.

Reply:
Ha, good point.

Reply:
fixed it